spec/payload/putMaintenanceConfig.json

```diff
@@ -0,0 +1,21 @@
+{
+    "id": "maintenanceConfigID",
+    "name": "sampleMaintenanceConfig",
+    "query": "entity.application.id:\"appID\" OR entity.application.id:\"anotherAppID\" ",
+    "scheduling": {
+        "start": 1683827571245,
+        "duration": {
+            "amount": 2,
+            "unit": "HOURS"
+        },
+        "type": "ONE_TIME"
```

Comment:
Can we have a payload for recurrence (maybe use `FREQ=WEEKLY;INTERVAL=2;BYDAY=SA;COUNT=10` for the RRULE)